.github/workflows/quickstart_http_custom_mechanism_ci.yml

```diff
@@ -0,0 +1,17 @@
+name: WildFly http_custom_mechanism Quickstart CI
```

Comment:
this file should be named instead quickstart_http-custom-mechanism_ci.yml

Reply:
FIxed!